src/gui/dialog/SettingsDialog.h

```diff
@@ -48,8 +50,12 @@ class SettingsDialog : public GladeGui
 	XOJ_TYPE_ATTRIB;
 
 	Settings* settings;
+	Control* control;
 	GtkWidget* callib;
 	int dpi;
+	std::vector<DeviceInfo> audioInputDevices;
```

Comment:
for vector and string we have the using std::vector in the header already, so this is not needed, as these classes are used very often, but also, no issue...